util.js

```diff
@@ -32,3 +32,19 @@ exports.runSpawn = function runSpawn({ cwd } = {}) {
     })
   }
 }
+
+exports.runSpawn = runSpawn
+
+exports.tagVersionInGit = async function (version) {
+  const run = runSpawn()
+
+  await run('git', ['push', 'origin', `:refs/tags/${version}`])
+  await run('git', [
+    'tag',
+    '-fa',
+    `"${version}"`,
+    '-m',
+    `"Update tag ${version}"`,
```

Comment:
it would be useful if you could explain why you've chosen to provide these flags, it is not obvious otherwise.

my doubts are:

- why do we need -a?
- why do we need -m?

Reply:
The flags -a and -m are needed to make it an annotated tag with a message. 
They're not needed if we want to leave the default tag message ([OPTIC-RELEASE-AUTOMATION] release/vX.X.X)